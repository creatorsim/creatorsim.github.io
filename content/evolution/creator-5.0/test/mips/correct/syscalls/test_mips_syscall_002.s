#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:
	# print int
	li $v0, 2
	li.s $f12, 6673
	syscall

  	# print negative float
    li.s $f12, -86.974
    syscall
    
    li.s $f12, 1.1
    syscall
    