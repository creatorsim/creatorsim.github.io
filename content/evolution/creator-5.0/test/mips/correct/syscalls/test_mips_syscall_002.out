
6673
-86.9739990234375
1.100000023841858
cr[PC]:0x34; ir[1,at]:0x3f8ccccd; ir[2,v0]:0x2; sfpr[f12]:0x3F8CCCCD; dfpr[FP12]:0x3F8CCCCD00000000; keyboard[0x0]:''; display[0x0]:'6673-86.97399902343751.100000023841858'; 

